### ISSUE TITLE
German translation

### ISSUE THREAD

**Issue by thedoorsfanatic** [amcolash/MMM-GoogleFit#24]
Hi,
is there a way to translate the module into German?
Like the weekdays and everything else?
If you need help regarding the translated text, I can help you out but I am a newbie with no programming skills

**Comment by amcolash** [comment]
Yes, I could look into adding in translations as MM supports it. Right now I have a list of days in English:
`"S", "M", "T", "W", "T", "F", "S"`

Looking at a list of German days of the week, it looks like there might be some day collisions with repeated letters like in English
Montag (Monday)
Dienstag (Tuesday)
Mittwoch (Wednesday)
Donnerstag (Thursday)
Freitag (Friday)
Samstag (Saturday)
Sonntag (Sunday)

so it would look like `"M", "D", "M", "D", "F", "S", "S"`

Is the first day of the week normally Monday/Montag on calendars? I can also add that if needed. Not sure if there is much else for translations - let me know if I missed things

**Comment by thedoorsfanatic** [comment]
Hi amoclash,

thanks for your quick reply. At the time I am using the code from More settings added #22 by ferferga which looks nice and works for me
![DSC_0189_5](https://user-images.githubusercontent.com/47753193/59015016-447e7980-883e-11e9-82f0-433715096ef1.JPG)

I also would like the kg in decimals and would translate all the messages from your code in German.
In Germany there are some calendars starting with Sonntag, others with Montag so I would leave this option to the user. anyway, great work on the plugin. I wish I could do that since I have many ideas and would love to program a Garmin connect module
